doc/whatsnew.rst

```diff
@@ -30,6 +30,8 @@ v0.2.8.dev
 
 - Add new function :func:`pyriemann.datasets.make_gaussian_blobs` for generating random datasets with SPD matrices
 
+- Add an example on ERP visualization
```

Comment:
you also added a public function that should be documented in the public API page.